crates/turborepo-lib/src/commands/link.rs

```diff
@@ -89,11 +89,9 @@ pub(crate) enum SelectedSpace<'a> {
     Space(&'a Space),
 }
 
-pub(crate) const REMOTE_CACHING_INFO: &str = "  Remote Caching shares your cached Turborepo task \
-                                              outputs and logs across
-  all your team’s Vercel projects. It also can share outputs
-  with other services that enable Remote Caching, like CI/CD systems.
-  This results in faster build times and deployments for your team.";
+pub(crate) const REMOTE_CACHING_INFO: &str =
+    "Remote Caching makes your caching multiplayer,\nsharing build outputs and logs between \
```

Comment:
can we not repeat "caching" twice if we're going to change the language? how about:

```suggestion
    "Remote Caching makes your tasks multiplayer,\nsharing build outputs and logs between \
```

Reply:
Is `task caching` better? I get that that doesn't make it not say "caching" twice but `tasks` leaves room for interpretation.

Reply:
🤷🏾 I don't like task caching that much better, but not going to block on it either!